showkase-processor/src/main/java/com/airbnb/android/showkase/processor/writer/WriterUtils.kt

```diff
@@ -105,18 +105,30 @@ internal fun CodeBlock.Builder.closeOrContinueListCodeBlock(
 }
 
 internal fun CodeBlock.Builder.addShowkaseBrowserComponent(
-    showkaseMetadata: ShowkaseMetadata.Component
+    showkaseMetadata: ShowkaseMetadata.Component,
+    isParameterComponent: Boolean
 ) {
+    var componentKey = ("${showkaseMetadata.packageName}" +
+            "_${showkaseMetadata.enclosingClass}" +
+            "_${showkaseMetadata.showkaseGroup}" +
+            "_${showkaseMetadata.showkaseName}").replace(
+        "\\s".toRegex(),
+        ""
+    )
+    if (isParameterComponent) {
+        componentKey += "$" + "index"
```

Comment:
```suggestion
        componentKey += "$index"
```

Reply:
would need to escape the $ if you go this route